### PR TITLE
Add a button to configure and access the web application.

### PR DESCRIPTION
**Pull request by stephdl**
Introduce new translations for SOGo webmail and configuration messages, and add a button in the status page to configure and access the web application.

Capture d’écran du 2024-11-12 10-45-10

Capture d’écran du 2024-11-12 10-44-39

https://github.com/NethServer/dev/issues/7108

### CODE REVIEW

**Comment by stephdl** [comment]
cc @andre8244 can you review please

File: ui/public/i18n/en/translation.json [hunk 1]
```diff
@@ -18,7 +18,11 @@
     "mount": "Mount",
     "no_services": "No services",
     "no_images": "No images",
-    "no_volumes": "No volumes"
+    "no_volumes": "No volumes",
+    "not_configured": "Not configured",
+    "sogo_webapp": "SOGo webmail",
+    "open_webapp": "Open app",
```
Comment (andre8244) [inline ui/public/i18n/en/translation.json]:
```suggestion
    "open_webapp": "Open SOGo",
```
Let's make it uniform to Nextcloud and other webapps

File: ui/src/views/Status.vue [hunk 2]
```diff
@@ -39,6 +39,42 @@
...
+              :disabled="loading.getConfiguration"
+              :icon="ArrowRight20"
+              @click="goToAppPage(instanceName, 'settings')"
+            >
+              {{ $t("status.configure") }}
+            </NsButton>
+          </template>
+        </NsInfoCard>
+      </cv-column>
+    </cv-row>
```
Comment (andre8244) [inline ui/src/views/Status.vue]: Can we put the new card on the same row as the others?

File: ui/src/views/Status.vue [hunk 3]
```diff
@@ -343,10 +382,61 @@ export default {
...
+    goToWebapp() {
+      window.open(`https://${this.host}`, "_blank");
+    },
+    async getConfiguration() {
+      this.loading.getConfiguration = true;
+      this.error.getConfiguration = "";
+      const taskAction = "get-configuration";
+
+      // register to task error
+      this.core.$root.$off(taskAction + "-aborted");
```
Comment (andre8244) [inline ui/src/views/Status.vue]: Let's use `eventId` variable, like here: https://github.com/NethServer/ns8-core/blob/main/core/ui/src/components/domains/ChangeUserPasswordModal.vue#L127

**Comment by stephdl** [comment]
should be good andre

**Review by andre8244 — APPROVED** [review]

**Pull request merged**